test/compiler/AbstractInterpreter.jl

```diff
@@ -45,6 +45,14 @@ import Base.Experimental: @MethodTable, @overlay
 @MethodTable(OverlayedMT)
 CC.method_table(interp::MTOverlayInterp) = CC.OverlayMethodTable(CC.get_world_counter(interp), OverlayedMT)
 
+function CC.add_remark!(interp::MTOverlayInterp, ::CC.InferenceState, remark)
+    if interp.meta !== nothing
```

Comment:
In 1.9 the meta field is missing

Reply:
The meta field was introduced in 1d8f7e0723d e.g. #41199 which we didn't backport to 1.9

Reply:
fix incoming.